### ISSUE TITLE
Alternative text not being displayed on Android

### ISSUE THREAD

**Issue by RafaelFernandesBR** [laurent22/joplin#12558]
### Operating system

Android

### Joplin version

3.3.8

### Current behaviour

It is not being displayed on android, the alternative text of an image, shows an id like 003abc1694024d09b6b9c187bbc4a02d

### Expected behaviour

It should show the image's alt text, when there is one.
Example:
```markdown
![Description](image_path)
```

**Comment by personalizedrefrigerator** [comment]
Thank you for reporting this! I'm summarizing what I've found while looking into the issue:

### Summary

This seems to be [an upstream Chromium issue](https://issues.chromium.org/issues/418962695#comment10) that has recently been marked as resolved.

### Additional details
 
- I can reproduce this on my Android device.
    - Version info: Android 13 with WebView 137.0.7151.89, Joplin 3.4.0 (dev build), and TalkBack 13.5.
- Joplin uses a Chromium-based WebView for its note viewer.
- The [relevant Chromium bug](https://issues.chromium.org/issues/418962695) seems very similar or identical to this issue. [At least one user](https://issues.chromium.org/issues/418962695#comment10) has confirmed that the upstream issue affects them in various apps.
   - The issue tracker states that the issue should be fixed in a prerelease version of Chrome/WebView.

### Observations

- The ALT text **does** seem to be set. If I connect to the note viewer from the Chrome DevTools, I see that the relevant image has `alt="a large rectangle"`:
  ![screenshot, chrome dev tools with accessibility panel open](https://github.com/user-attachments/assets/24e7c3f8-c5a9-4a23-8840-f35bd324c775)
  
  <details><summary>Screenshot description</summary>

  The above screenshot shows:
  - The "Accessibility" panel in the dev tools — the `img` has the following computed properties:
      - (Dropdown) name: "a large rectangle"
         - aria-labelledby: Not specified
         - aria-label: Not specified
         - alt: "a large rectangle"
         - title: Not specified
         - Role: image
         - url: file:///data/user/0/net.cozic.joplin/files/3362d...

   - The "Elements" panel shows that the `<img` tag has `alt="a large rectangle"` and a `title` attribute without a value.
   - The main viewer panel highlights the single image in the note. 
   </details>
- I can reproduce this issue on Android 13 with WebView 137.0.7151.89, Joplin 3.4.0 (dev build), and TalkBack 13.5.

**Edited**: Reorganized, simplified summary.

**Comment by personalizedrefrigerator** [comment]
The prerelease version (v139.0.7245.0) of the Android System WebView app fixes the issue for me.

<details><summary>Testing steps</summary>

1. Installed "Android System WebView Canary" from the Play Store.
2. After installing, clicked the "Open" button (just before "Uninstall"). This opened the "WebView DevTools".
3. In the WebView dev tools, I clicked "Change provider".
    - Change provider was shown in a message box with label: "Android System WebView Canary 139.0.7245.0 is not the system's currently selected WebView provider".
4. Selected the "Canary" WebView in the dialog that was shown.
   - **Note**: I needed to remove a secondary user account — according to the dialog, the WebView needs to be installed for all users.
5. Re-launched Joplin.
6. Re-opened the test note.
7. Verified that TalkBack reads the ALT text for the test image (rather than the file name). 

</details>

**Comment by personalizedrefrigerator** [comment]
Since it's less likely to be stable than the standard system WebView, I'm not sure I would recommend using the Canary WebView version.

Would a workaround in a Joplin plugin be helpful (for until the upstream bug is fixed)? For example, a "Replace images with ALT text" setting could be added to the [Extra Markdown Viewer Settings](https://github.com/personalizedrefrigerator/joplin-plugin-extra-viewer-settings/) plugin.
